app/cdash/app/Model/CoverageFile2User.php

```diff
@@ -289,31 +291,18 @@ public function GetCoverageFileId($buildid): int|false
     /**
      * Get the list of authors for the project
      *
-     * @return array<int>|false
+     * @return array<int>
      */
-    public function GetUsersFromProject(): array|false
+    public function GetUsersFromProject(): array
     {
         if (!isset($this->ProjectId) || $this->ProjectId < 1) {
             abort(500, 'CoverageFile2User:GetUsersFromProject: projectid not valid');
         }
 
-        $db = Database::getInstance();
-        $query_result = $db->executePrepared('
-                            SELECT DISTINCT userid
-                            FROM coveragefile2user, coveragefilepriority
-                            WHERE
-                                coveragefilepriority.id=coveragefile2user.fileid
-                                AND coveragefilepriority.projectid=?
-                        ', [intval($this->ProjectId)]);
-        if (empty($query_result)) {
-            add_last_sql_error('CoverageFile2User:GetUsersFromProject');
-            return false;
-        }
-        $userids = [];
-        foreach ($query_result as $query_array) {
-            $userids[] = intval($query_array['userid']);
-        }
-        return $userids;
+        return DB::table('coveragefilepriority')
+                 ->join('coveragefile2user', 'coveragefilepriority.id', '=', 'coveragefile2user.fileid')
+                 ->where('coveragefilepriority.projectid', '=', intval($this->ProjectId))
+                 ->pluck('userid')->toArray();
```

Comment:
Do we need to add a `DISTINCT` clause to this query to reduce the size of the result array?  It's in the original query, but I'm not sure if it's necessary or not.